test/unit-tests/kitmanager.test.ts

```diff
@@ -1,7 +1,8 @@
+/* eslint-disable no-unused-expressions */
 import {readKitsFile, getShellScriptEnvironment} from '@cmt/kit';
 import {expect} from '@test/util';
 import * as path from 'path';
-import * as vscode from 'vscode';
+//import * as vscode from 'vscode';
```

Comment:
Minor. If we don't need vscode anymore, let's remove instead of leave it commented out.